search/vertex-search-options/vertex_search_options.ipynb

```diff
@@ -0,0 +1,649 @@
+        "    \"max_output_tokens\": 8192,\n",
+        "    \"temperature\": 1,\n",
+        "    \"top_p\": 0.95,\n",
+        "}\n",
+        "\n",
+        "safety_settings = [\n",
+        "    SafetySetting(\n",
+        "        category=SafetySetting.HarmCategory.HARM_CATEGORY_HATE_SPEECH,\n",
+        "        threshold=SafetySetting.HarmBlockThreshold.BLOCK_ONLY_HIGH,\n",
+        "    ),\n",
+        "    SafetySetting(\n",
+        "        category=SafetySetting.HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT,\n",
+        "        threshold=SafetySetting.HarmBlockThreshold.BLOCK_ONLY_HIGH,\n",
+        "    ),\n",
+        "    SafetySetting(\n",
+        "        category=SafetySetting.HarmCategory.HARM_CATEGORY_SEXUALLY_EXPLICIT,\n",
+        "        threshold=SafetySetting.HarmBlockThreshold.BLOCK_ONLY_HIGH,\n",
+        "    ),\n",
+        "    SafetySetting(\n",
+        "        category=SafetySetting.HarmCategory.HARM_CATEGORY_HARASSMENT,\n",
+        "        threshold=SafetySetting.HarmBlockThreshold.BLOCK_ONLY_HIGH,\n",
+        "    ),\n",
+        "]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "jCJAOvHdLrGI"
+      },
+      "outputs": [],
+      "source": [
+        "def generate():\n",
+        "    model = GenerativeModel(\n",
+        "        \"gemini-1.5-pro-001\", tools=tools, system_instruction=[system_instructions]\n",
+        "    )\n",
+        "\n",
+        "    response = model.generate_content(\n",
+        "        [SEARCH_QUERY],\n",
+        "        generation_config=generation_config,\n",
+        "        safety_settings=safety_settings,\n",
+        "        stream=True,\n",
+        "    )\n",
+        "\n",
+        "    return response"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "jLN2tz5dM4jp"
+      },
+      "outputs": [],
+      "source": [
+        "response = generate()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "BNrZ-w3gRoLx"
+      },
+      "outputs": [],
+      "source": [
+        "for candidate in response:\n",
+        "    print(candidate)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lQv61qA5Q71g"
+      },
+      "source": [
+        "##### [Here](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/grounding/intro-grounding-gemini.ipynb) is another Grounding Gemini with Vertex AI Search Example which you might find useful"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "V2X81IQUOImp"
+      },
+      "source": [
+        "## Part 3: Vertex AI Search with LangChain\n",
+        "\n",
+        "Leverage Vertex AI Search as [LangChain Retriever](https://python.langchain.com/v0.2/docs/integrations/retrievers/google_vertex_ai_search/)\n"
```

Comment:
Maybe add a little more information about what this can be used for? Part of larger RAG workflows?